packages/react-error-overlay/src/utils/unmapper.js

```diff
@@ -4,6 +4,12 @@ import { getSourceMap } from './getSourceMap';
 import { getLinesAround } from './getLinesAround';
 import path from 'path';
 
+function count(search: string, string: string): number {
+  let count = -1, index = 0;
+  for (; index !== -1; count++, (index = string.indexOf(search, index + 1)));
```

Comment:
It would be nice to rewrite this in a more readable way. It's fine to be more verbose.